README.md

```diff
@@ -0,0 +1,18 @@
+openmrs-config-ces
+==============================
+
+#### Usage
+
+`mvn clean compile` - Will generate your configurations into "target/openmrs-packager-config/configuration"
+`mvn clean package` - Will compile as above, and generate a zip package at "target/${artifactId}-${version}.zip"
+
+In order to facilitate deploying configurations easily into an OpenMRS SDK server, one can add an additional parameter
+to either of the above commands to specify that the compiled configuration should also be copied to an existing 
+OpenMRS SDK server:
+
+`mvn clean compile -DserverId=wellbody` - Will compile as above, and copy the resulting configuration to `~/openmrs/wellbody/configuration`
```

Comment:
Why copy? Why not symlink? Copies merely introduce additional, well, copies of the same thing that can come out of sync, and about which you have to ask, "What version is this on? When was this updated?"

Reply:
Because that's how the plugin works :)  We can discuss, but this is also how the OpenMRS SDK operates  Copying is my preferred method, I think it is cleaner.  We can discuss though.  Let's take that discussion onto the plugin repository.

@mogoodrich - here, I'd recommend using "-DserverId=ces" as an example.

Reply:
Agree with README change, just changed wellbody to CES

Reply:
I think copying is cleaner as well... generally duplication is bad, but this is more of an explicity deployment step, which I think is better.

Also, practically, a "mvn clean" would clean out the build directory and if you symlink that would mean you'd break your dev environment every time you wanted to clean out your existing build... bottom line, "building" and "deploying to my dev environment" seem like two separate steps to me.

It's true that we may want to support some sort of symlinking for hot reloading of things like html forms...  in the current module you'd need to run a compile every time you change a form... maybe the answer here would be the equivalent of a "watch" that triggers a compile on a file change?